### ISSUE TITLE
Does hangfire dashboard supports jobs running in multiple processes

### ISSUE THREAD

**Issue by hnarisetty** [HangfireIO/Hangfire#1609]
I have below scenario and I would like to know if hangfire supports this or not. I have two processes running different jobs pointing to the same SQL Server Hangfire database.

Process A ( DownloadOrdersJob)
Process B ( UnlockOrdersJob)
These two are two different jobs and we want to run in two different processes. I am pointing to the same hangfire database. When I access the hangfire dashboard, http://xxxx:9090/Hangfire , I see both the jobs , but one is always with an error message as "Unable to load dll ...." . Is this scenario supported in Hangfire?

**Comment by pieceofsummer** [comment]
Yes it does, but it requires for processes and dashboard to share the jobs’ codebase (or at least the interface part of it), so it can resolve method signatures and attributes for those.

You can extract job interfaces into the shared assembly, while the actual implementations are kept private for the corresponding server processes.

**Comment by Sonic198** [comment]
> Yes it does, but it requires for processes and dashboard to share the jobs’ codebase (or at least the interface part of it), so it can resolve method signatures and attributes for those.
> 
> You can extract job interfaces into the shared assembly, while the actual implementations are kept private for the corresponding server processes.

@pieceofsummer thank you for explaining that scenario. How recurring jobs should be registered then? With interface `RecurringJob.AddOrUpdate<ISampleJob>(...` or with implementation class `RecurringJob.AddOrUpdate<SampleJob>(...` or it really doesn't matter?
